src/NHibernate.Test/Ado/GenericBatchingBatcherFixture.cs

```diff
@@ -29,7 +29,8 @@ protected override bool AppliesTo(Dialect.Dialect dialect)
 			return !(dialect is FirebirdDialect) &&
 			       !(dialect is Oracle8iDialect) &&
 			       !(dialect is MsSqlCeDialect) &&
-			       !(dialect is HanaDialectBase);
+			       !(dialect is HanaDialectBase) &&
+			       !(dialect is SybaseSQLAnywhere10Dialect);
```

Comment:
This batcher is not supported by SQL Anywhere. It seems it does not supports to have multiple inserts in the same query, as described [here](https://stackoverflow.com/a/4400607/1178314).